package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "heroprotocol",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

Comment:
Make it 0.4.0, and note in the README and CHANGELOG that last version you can fully test with.

And that's a go!